apps/wallet/tests/utils/auth.ts

```diff
@@ -23,10 +23,20 @@ export async function importWallet(page: Page, extensionUrl: string, mnemonic: s
     await page.goto(extensionUrl, { waitUntil: 'commit' });
     await page.getByRole('button', { name: /Add Profile/ }).click({ timeout: SHORT_TIMEOUT });
     await page.getByText('Mnemonic', { exact: true }).click();
-    await page
-        .getByPlaceholder('Word')
-        .first()
-        .fill(typeof mnemonic === 'string' ? mnemonic : mnemonic.join(' '));
+
+    const mnemonicArray = typeof mnemonic === 'string' ? mnemonic.split(' ') : mnemonic;
```

Comment:
This reminds me that we should probably add a test to `apps/wallet/tests/onboarding.spec.ts` so that we test `import wallet with 12 words mnemonic` & `import wallet with 24 words mnemonic`